Ramda/Extensions/head.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  head.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 27/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Returns the first element of the given string.
+
+     - parameter value: String to check
+
+     - returns: The first element of the given string.
+
+     */
+    public class func head(_ string: String) -> String {
+        return string.first == nil ? "" : String(string.first!)
```

Comment:
We avoid force unwrapping. 

```
guard let head = string.first else {
    return ""
}

return String(head)
```

Reply:
Also should we not return a nil if the string is empty or is the correct way to return an empty string?

Reply:
The correct way is to return an empty string